NEWS.md

```
@@ -1,3 +1,6 @@
+## igvShiny 1.9.33
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)